unlock-app/src/components/interface/checkout/main/Select.tsx

```diff
@@ -431,13 +431,68 @@ export function Select({ checkoutService }: Props) {
     isLoading,
   ])
 
+  useEffect(() => {
+    const selectedLock = searchParams.get('selectedLock')
+    const privyOAuthState = searchParams.get('privy_oauth_state')
+    const privyOAuthProvider = searchParams.get('privy_oauth_provider')
+    const privyOAuthCode = searchParams.get('privy_oauth_code')
+
+    if (
+      selectedLock &&
+      privyOAuthState &&
+      privyOAuthProvider === 'google' &&
```

Comment:
I think we should not care about whether it is Google or something else IMO.

Reply:
sure!